docs/users/security/safu.md

```diff
@@ -0,0 +1,92 @@
+<!--
+order: 3
+-->
+
+# Simple Arrangement for Funding Upload (SAFU)
+
+Learn about the Simple Arrangement for Funding Upload (SAFU)
+on Evmos {synopsis}
+
+## Overview
+
+The [Simple Arrangement for Funding Upload (the "SAFU")](https://docs.google.com/document/d/1kyPn-uRQnCOjeCHjH6IRGGDOTMVHMLeQXeJql9MXnkw/edit#heading=h.vcatw1yk8om7)
```

Comment:
Can you download the doc in PDF and upload it to Github instead?

Reply:
@fedekunze yes, there are a few corrections to be made tho, like the dropbox address and reward equation